jerry-ext/module/module.c

```diff
@@ -202,29 +214,51 @@ jerryx_module_native_resolver (const jerry_char_t *name, /**< name of the module
  *   - an error indicating that something went wrong during the attempt to load the module.
  */
 jerry_value_t
-jerryx_module_resolve (const jerry_char_t *name, /**< name of the module to load */
-                       const jerryx_module_resolver_t *resolvers_p, /**< list of resolvers */
+jerryx_module_resolve (const jerry_value_t name, /**< name of the module to load */
+                       const jerryx_module_resolver_t **resolvers_p, /**< list of resolvers */
```

Comment:
`**` ? A pointer to a list of pointers which points to the resolvers? Why not an array of resolvers?

Reply:
Because the resolvers are stored in-contiguously, and we do not want to make copies of them, right?

Reply:
Are they used multiple times? Or why they are scattered?

I think one application use one resolver list, but this is not much of a code size increase, so lleave it be.

Reply:
Resolvers may be declared global in separate .c files (jerryx_module_native_resolver` certainly is), and they are collected into an array only once - the place that calls `jerryx_module_resolve ()`.